src/stream/stream/mod.rs

```diff
@@ -51,6 +52,7 @@ use fold::FoldFuture;
 use min_by::MinByFuture;
 use next::NextFuture;
 use nth::NthFuture;
+use skip_while::SkipWhile;
```

Comment:
Let's export `SkipWhile` (see lines 40-43 above).

The general policy is that we don't re-export `Future` types because functions that return them will eventually become `async fn`s, while we do export `Stream` types.

Reply:
ah, i see now, makes sense.